webui/module/Analytics/view/analytics/analytics/configuration.phtml

```diff
@@ -0,0 +1,377 @@
+    var nodeclass = "node node_" + d.id;
+    return nodeclass;
+  })
+  .attr("id", d => d.id)
+  .attr("r", 8)
+  .attr("fill", function(d, i) {
+    return getColorForResourceType(d.type);
+  })
+  .style("fill-opacity", 0.5);
+
+var label = node.append("text")
+  .text(d => d.name)
+  .attr("class", function(d) {
+    var labelclass = "nodelabel nodelabel_" + d.id;
+    return labelclass;
+  })
+  .attr("fill", "#999999")
+  .attr("font-size", 10)
+  .attr("font-family", "Helvetica")
+  .attr("x", 15)
+  .attr("y", 15)
+  .attr("dx", 20)
+  .attr("dy", 0);
+
+node.append("title")
+  .text(d => d.name);
+
+circle.on("mouseover", function(d) {
+
+    d3.selectAll(".node")
+      .attr("r", 8)
+      .style("stroke", "#ffffff")
+      .style("stroke-width", 0)
+      .style("stroke-opacity", 0)
+      .style("fill-opacity", 0.5)
+    d3.selectAll(".line")
+      .attr("stroke", "#999999")
+      .attr("stroke-width", 1)
+      .attr("stroke-opacity", 0.25)
+    d3.selectAll(".nodelabel")
+      .attr("font-size", 10)
+      .attr("font-weight", "normal")
+      .attr("fill", "#999999")
+    d3.selectAll(".nodelabel_" + d.target.id)
+      .attr("font-size", 14)
+      .attr("font-weight", "bold")
+      .attr("fill", "#000000")
+      .style("fill-opacity", 1)
+    d3.select(this)
+      .attr("r", 16)
+      .style("stroke", "#cccccc")
+      .style("stroke-width", 8)
+      .style("stroke-opacity", 0.5)
+      .style("fill-opacity", 1)
+
+    for (var i = 0; i < links.length; i++) {
+
+      if (links[i].target.id === d.target.id) {
+        console.log(d.target.id)
+        d3.selectAll(".node_" + links[i].source.id)
+          .attr("r", 16)
+          .style("stroke", "#cccccc")
+          .style("stroke-width", 8)
+          .style("stroke-opacity", 0.5)
+          .style("fill-opacity", 1)
+        d3.selectAll(".target_" + links[i].target.id)
+          .attr("stroke-width", 1)
+          .attr("stroke-opacity", 1)
+          .attr("stroke", "#000000")
+        d3.selectAll(".nodelabel_" + links[i].source.id)
+          .attr("font-size", 14)
+          .attr("font-weight", "bold")
+          .attr("fill", "#000000")
+      }
+      if (links[i].source.id === d.target.id) {
+        d3.selectAll(".node_" + links[i].target.id)
+          .attr("r", 16)
+          .style("stroke", "#cccccc")
+          .style("stroke-width", 8)
+          .style("stroke-opacity", 0.5)
+          .style("fill-opacity", 1)
+        d3.selectAll(".source_" + links[i].source.id)
+          .attr("stroke-width", 1)
+          .attr("stroke-opacity", 1)
+          .attr("stroke", "#000000")
+        d3.selectAll(".nodelabel_" + links[i].target.id)
+          .attr("font-size", 14)
+          .attr("font-weight", "bold")
+          .attr("fill", "#000000")
+      }
+    }
+
+    const node = nodes.find(node => node.id === d.target.id);
+
+    if (node) {
+        d3.select("#resource")
+            .html(escapeHtml(JSON.stringify(node.resource, null, "  ")));
```

Comment:
```suggestion
            .text(JSON.stringify(node.resource, null, "  "));
```
It turns out, that escapeHtml is not required. Hopefully I this way it is also more generic.

Reply:
applied